vatvalidate/validate.py

```diff
@@ -0,0 +1,50 @@
+def _modulus_9755(vat_digits: list[int], use_9755: bool = False) -> bool:
```

Comment:
I'm intrigued what the tax authorities find this constraint on VAT codes useful for...